services/txt2img.py

```diff
@@ -16,7 +16,6 @@
 sd_pipeline = None
 
 def generate_txt2img(args):
-    print(json.dumps(args))
```

Comment:
it would be nice to still log in debug mode. i think that's if [if app.debug:] and adding app to the things we import from flask

Reply:
Added a --verbose / -v flag that will log args when set

Reply:
Done, ready for review

Reply:
probably want the same thing in generate_img2img, as well.

might be better to have a flask global for the verbose mode instead of plumbing through all the calls